elasticsearch-quickstart/README.md

```diff
@@ -0,0 +1,52 @@
+# elasticsearch-quickstart
```

Comment:
Other quickstarts I'm familiar with have a different structure, see for example https://github.com/quarkusio/quarkus-quickstarts/blob/d8d6dbbb49fa2357b6124aad45eee4b3dc1fd061/hibernate-orm-quickstart/README.md#L9